src/app.ts

```diff
@@ -64,8 +64,10 @@ export default class App {
     app.set('layout', 'layouts/layout.ejs');
 
     app.use(express.static(path.join(__dirname)));
-
-    app.get('/', (_req, res) => res.render('pages/index.ejs'));
+    const inspectletKey = ConfigService.getStringValue('inspectlet.wid');
+    app.get('/', (_req, res) => {
+      res.render('pages/index.ejs', { inspectlet_key: inspectletKey || '' });
+    });
```

Comment:
As `inspectlet.wid` is actually an optional property, this won't be always present, right? If yes, the application would fail to start if this is not present.

Suggesting to change this to:

```js
const inspectletKey = ConfigService.hasValue('inspectlet.wid') ? ConfigService.getStringValue('inspectlet.wid') : '';
app.get('/', (_req, res) => res.render('pages/index.ejs', { inspectlet_key: inspectletKey }));
```

Support for `hasValue` was recently added, you might have to pull and merge latest from `main`.